### ISSUE TITLE
Crash when place/break stabilized spawner ..

### ISSUE THREAD

**Issue by kuba12345** [brandon3055/Draconic-Evolution#902]
## Description
Error occurs when place/break stabilized spawner ... seems like problem with spawning or something with iguana tinker tweaks...
## Base information
* Minecraft version: 1.7.10
* Minecraft Forge version: 1614
* Mod Pack: Custom

## Crash report
```
---- Minecraft Crash Report ----
// Uh... Did I do that?

Time: 9.9.17 14:48
Description: Exception ticking world

java.lang.ClassCastException: iguanaman.iguanatweakstconstruct.mobheads.tileentities.IguanaSkullTileEntity cannot be cast to cofh.thermaldynamics.block.TileTDBase
	at cofh.thermaldynamics.block.BlockTDBase.isSideSolid(BlockTDBase.java:52)
	at net.minecraft.world.World.func_147466_a(World.java:2627)
	at net.minecraft.world.SpawnerAnimals.func_77190_a(SpawnerAnimals.java:215)
	at net.minecraft.world.SpawnerAnimals.func_77192_a(SpawnerAnimals.java:130)
	at net.minecraft.world.WorldServer.func_72835_b(WorldServer.java:161)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:625)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at cofh.thermaldynamics.block.BlockTDBase.isSideSolid(BlockTDBase.java:52)
	at net.minecraft.world.World.func_147466_a(World.java:2627)
	at net.minecraft.world.SpawnerAnimals.func_77190_a(SpawnerAnimals.java:215)
	at net.minecraft.world.SpawnerAnimals.func_77192_a(SpawnerAnimals.java:130)
	at net.minecraft.world.WorldServer.func_72835_b(WorldServer.java:161)

-- Affected level --
Details:
	Level name: world
	All players: 1 total; [EntityPlayerMP['kubajuk'/1593, l='world', x=-186,20, y=53,00, z=135,87]]
	Chunk stats: ServerChunkCache: 443 Drop: 2
	Level seed: -776765044144183177
	Level generator: ID 00 - default, ver 1. Features enabled: false
	Level generator options: 
	Level spawn location: World: (0,0,0), Chunk: (at 0,0,0 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 0 game time, 0 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: survival (ID 0). Hardcore: false. Cheats: false
Stacktrace:
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:625)
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_121, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 342307896 bytes (326 MB) / 1250951168 bytes (1193 MB) up to 1908932608 bytes (1820 MB)
	JVM Flags: 1 total; -Xmx2048M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 12, tallocated: 96
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1614 101 mods loaded, 90 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.4.1614-1.7.10-universal.jar) 
	UCHIJAAAA	Forge{10.13.4.1614} [Minecraft Forge] (forge-1.7.10-10.13.4.1614-1.7.10-universal.jar) 
	UCHIJAAAA	appliedenergistics2-core{rv3-beta-6} [Applied Energistics 2 Core] (minecraft.jar) 
	UCHIJAAAA	CodeChickenCore{1.0.7.47} [CodeChicken Core] (minecraft.jar) 
	UCHIJAAAA	Micdoodlecore{} [Micdoodle8 Core] (minecraft.jar) 
	UCHIJAAAA	NotEnoughItems{1.0.5.120} [Not Enough Items] (NotEnoughItems-1.7.10-1.0.5.120-universal.jar) 
	UCHIJAAAA	PlayerAPI{1.4} [Player API] (minecraft.jar) 
	UCHIJAAAA	technomCore{0.12.5} [Technomancy Core] (minecraft.jar) 
	UCHIJAAAA	OpenModsCore{0.10} [OpenModsCore] (minecraft.jar) 
	UCHIJAAAA	<CoFH ASM>{000} [CoFH ASM] (minecraft.jar) 
	UCHIJAAAA	appliedenergistics2{rv3-beta-6} [Applied Energistics 2] (appliedenergistics2-rv3-beta-6.jar) 
	UCHIJAAAA	bdlib{1.9.4.109} [BD Lib] (bdlib-1.9.4.109-mc1.7.10.jar) 
	UCHIJAAAA	ae2stuff{0.5.1.9} [AE2 Stuff] (ae2stuff-rv3-0.5.1.9-mc1.7.10.jar) 
	UCHIJAAAA	AWWayofTime{v1.3.3} [Blood Magic: Alchemical Wizardry] (BloodMagic-1.7.10-1.3.3-17.jar) 
	UCHIJAAAA	Baubles{1.0.1.10} [Baubles] (Baubles-1.7.10-1.0.1.10.jar) 
	UCHIJAAAA	Botania{r1.8-249} [Botania] (Botania r1.8-249.jar) 
	UCHIJAAAA	Avaritia{1.11} [Avaritia] (Avaritia-1.11.jar) 
	UCHIJAAAA	wanionlib{1.7.10-1.2} [WanionLib] (WanionLib-1.7.10-1.2.jar) 
	UCHIJAAAA	avaritiaddons{1.4} [Avaritiaddons] (Avaritiaddons-1.4.jar) 
	UCHIJAAAA	Backpack{2.0.1} [Backpack] (Backpacks-Mod.jar) 
	UCHIJAAAA	betterquesting{2.0.233} [Better Questing] (BetterQuesting-2.0.233.jar) 
	UCHIJAAAA	BotaniaVisualizer{1.7.2-1.0a} [BotaniaVisualizer] (BotaniaVisualizer-1.7.2-1.0a.jar) 
	UCHIJAAAA	botanicenergistics{@version@} [Botanic-Energistics] (BotanicEnergistics-0.2.7.jar) 
	UCHIJAAAA	CoFHCore{1.7.10R3.1.4} [CoFH Core] (CoFHCore-[1.7.10]3.1.4-329.jar) 
	UCHIJAAAA	BrandonsCore{1.0.0.12} [Brandon's Core] (BrandonsCore-1.0.0.12.jar) 
	UCHIJAAAA	BuildCraft|Core{7.1.21} [BuildCraft] (buildcraft-7.1.21.jar) 
	UCHIJAAAA	BuildCraft|Builders{7.1.21} [BC Builders] (buildcraft-7.1.21.jar) 
	UCHIJAAAA	BuildCraft|Energy{7.1.21} [BC Energy] (buildcraft-7.1.21.jar) 
	UCHIJAAAA	BuildCraft|Factory{7.1.21} [BC Factory] (buildcraft-7.1.21.jar) 
	UCHIJAAAA	BuildCraft|Robotics{7.1.21} [BC Robotics] (buildcraft-7.1.21.jar) 
	UCHIJAAAA	BuildCraft|Silicon{7.1.21} [BC Silicon] (buildcraft-7.1.21.jar) 
	UCHIJAAAA	BuildCraft|Transport{7.1.21} [BC Transport] (buildcraft-7.1.21.jar) 
	UCHIJAAAA	CarpentersBlocks{3.3.8.1} [Carpenter's Blocks] (Carpenters-Blocks-Mod-1.7.10.jar) 
	UCHIJAAAA	ChickenChunks{1.3.4.19} [ChickenChunks] (ChickenChunks-1.7.10-1.3.4.19-universal (3).jar) 
	UCHIJAAAA	ThermalFoundation{1.7.10R1.2.6} [Thermal Foundation] (ThermalFoundation-[1.7.10]1.2.6-118.jar) 
	UCHIJAAAA	ThermalExpansion{1.7.10R4.1.5} [Thermal Expansion] (ThermalExpansion-[1.7.10]4.1.5-248.jar) 
	UCHIJAAAA	DraconicEvolution{1.0.2h} [Draconic Evolution] (Draconic-Evolution-1.7.10-1.0.2h.jar) 
	UCHIJAAAA	MineFactoryReloaded{1.7.10R2.8.1} [MineFactory Reloaded] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	UCHIJAAAA	ForgeMultipart{1.2.0.345} [Forge Multipart] (ForgeMultipart-1.7.10-1.2.0.345-universal.jar) 
	UCHIJAAAA	GalacticraftCore{3.0.12} [Galacticraft Core] (GalacticraftCore-1.7-3.0.12.501.jar) 
	UCHIJAAAA	Mekanism{9.1.0} [Mekanism] (Mekanism-1.7.10-9.1.0.281.jar) 
	UCHIJAAAA	EnderIO{1.7.10-2.2.8.381} [Ender IO] (EnderIO-1.7.10-2.2.8.381.jar) 
	UCHIJAAAA	EnderStorage{1.4.7.38} [EnderStorage] (EnderStorage-1.7.10-1.4.7.38-universal.jar) 
	UCHIJAAAA	Waila{1.5.10} [Waila] (Waila-1.5.10_1.7.10.jar) 
	UCHIJAAAA	extracells{2.3.14} [Extra Cells 2] (ExtraCells-1.7.10-2.3.14b197.jar) 
	UCHIJAAAA	GalacticraftMars{3.0.12} [Galacticraft Planets] (Galacticraft-Planets-1.7-3.0.12.501.jar) 
	UCHIJAAAA	ExtraPlanets{1.8.1} [Extra Planets] (ExtraPlanets-1.8.1.jar) 
	UCHIJAAAA	ExtraUtilities{1.2.12} [Extra Utilities] (extrautilities-1.2.12.jar) 
	UCHIJAAAA	ForgeEssentials{1.4.5} [Forge Essentials] (forgeessentials-1.7.10-1.4.6.1341-server.jar) 
	UCHIJAAAA	iChunUtil{4.2.3} [iChunUtil] (iChunUtil-4.2.3.jar) 
	UCHIJAAAA	GraviGun{4.0.0-beta} [GraviGun] (GravityGun-4.0.0-beta.jar) 
	UCHIJAAAA	guideapi{1.7.10-1.0.1-20} [Guide-API] (Guide-API-1.7.10-1.0.1-20.jar) 
	UCHIJAAAA	inventorytweaks{1.59-dev-152-cf6e263} [Inventory Tweaks] (InventoryTweaks-1.59-dev-152 (1).jar) 
	UCHIJAAAA	IronChest{6.0.60.741} [Iron Chest] (ironchest-1.7.10-6.0.60.741-universal.jar) 
	UCHIJAAAA	JABBA{1.2.2} [JABBA] (Jabba-1.2.2_1.7.10.jar) 
	UCHIJAAAA	LogisticsPipes{0.9.3.126} [Logistics Pipes] (logisticspipes-0.9.3.126.jar) 
	UCHIJAAAA	Mantle{1.7.10-0.3.2.jenkins191} [Mantle] (Mantle-1.7.10-0.3.2b.jar) 
	UCHIJAAAA	MekanismGenerators{9.1.0} [MekanismGenerators] (MekanismGenerators-1.7.10-9.1.0.281.jar) 
	UCHIJAAAA	MekanismTools{9.1.0} [MekanismTools] (MekanismTools-1.7.10-9.1.0.281.jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatAppliedEnergistics{1.7.10R2.8.1} [MFR Compat: Applied Energistics] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatBuildCraft{1.7.10R2.8.1} [MFR Compat: BuildCraft] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatForgeMicroblock{1.7.10R2.8.1} [MFR Compat: ForgeMicroblock] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatThermalExpansion{1.7.10R2.8.1} [MFR Compat: Thermal Expansion] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	UCHIJAAAA	TConstruct{1.7.10-1.8.8.build988} [Tinkers' Construct] (TConstruct-1.7.10-1.8.8 (1).jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatTConstruct{1.7.10R2.8.1} [MFR Compat: Tinkers' Construct] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	UCHIJAAAA	MineFactoryReloaded|CompatVanilla{1.7.10R2.8.1} [MFR Compat: Vanilla] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	UCHIJAAAA	MineTweaker3{3.0.10} [MineTweaker 3] (MineTweaker3-1.7.10-3.0.10B.jar) 
	UCHIJAAAA	MTRM{1.0} [MineTweakerRecipeMaker] (MineTweakerRecipeMaker-1.7.10-1.1.1.jar) 
	UCHIJAAAA	numina{1.7.10} [Numina] (Numina-0.4.1.105.jar) 
	UCHIJAAAA	powersuits{1.7.10-0.11.1.114} [MachineMuse's Modular Powersuits] (ModularPowersuits-1.7.10-0.11.1.114.jar) 
	UCHIJAAAA	NetherOres{1.7.10R2.3.1} [Nether Ores] (NetherOres-[1.7.10]2.3.1-22.jar) 
	UCHIJAAAA	NOVA{2015-10-09} [NOVA] (NOVA-2015-10-09-universal.jar) 
	UCHIJAAAA	OpenMods{0.10} [OpenMods] (OpenModsLib-1.7.10-0.10.jar) 
	UCHIJAAAA	OpenBlocks{1.6} [OpenBlocks] (OpenBlocks-1.7.10-1.6.jar) 
	UCHIJAAAA	p455w0rdslib{1.0.4} [p455w0rd's Library] (p455w0rdslib-1.7.10-1.0.4.jar) 
	UCHIJAAAA	PortalGun{4.0.0-beta-6} [PortalGun] (PortalGun-4.0.0-beta-6 (1).jar) 
	UCHIJAAAA	bq_rf{2.0.25} [RF Expansion] (RFExpansion-2.0.25.jar) 
	UCHIJAAAA	rftools{4.23} [RFTools] (rftools-4.23.jar) 
	UCHIJAAAA	bq_standard{2.0.130} [Standard Expansion] (StandardExpansion-2.0.130.jar) 
	UCHIJAAAA	StorageDrawers{1.7.10-1.10.9} [Storage Drawers] (StorageDrawers-1.7.10-1.10.9.jar) 
	UCHIJAAAA	supremelaser{1.0} [Supreme Laser] (supremelaser-1.0.1.jar) 
	UCHIJAAAA	ThermalDynamics{1.7.10R1.2.1} [Thermal Dynamics] (ThermalDynamics-[1.7.10]1.2.1-172.jar) 
	UCHIJAAAA	Torched{4.0.0} [Torched] (Torched-4.0.0.jar) 
	UCHIJAAAA	ae2wct{1.7.10-rv3-1.8.7.4b} [AE2 Wireless Crafting Terminal] (WirelessCraftingTerminal-1.7.10-rv3-1.8.7.4b.jar) 
	UCHIJAAAA	worldedit{6.1.1-SNAPSHOT} [WorldEdit] (worldedit-forge-mc1.7.10-6.1.1-SNAPSHOT-dist.jar) 
	UCHIJAAAA	McMultipart{1.2.0.345} [Minecraft Multipart Plugin] (ForgeMultipart-1.7.10-1.2.0.345-universal.jar) 
	UCHIJAAAA	IguanaTweaksTConstruct{1.7.10-2.1.6.163} [Iguana Tinker Tweaks] (IguanaTinkerTweaks-1.7.10-2.1.6.jar) 
	UCHIJAAAA	technom{0.12.5} [Technomancy] (Technomancy - 0.12.5 - 1.7.10.jar) 
	UCHIJAAAA	ForgeMicroblock{1.2.0.345} [Forge Microblocks] (ForgeMultipart-1.7.10-1.2.0.345-universal.jar) 
	UD	MineFactoryReloaded|CompatAtum{1.7.10R2.8.1} [MFR Compat: Atum] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	UD	MineFactoryReloaded|CompatBackTools{1.7.10R2.8.1} [MFR Compat: BackTools] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	UD	MineFactoryReloaded|CompatChococraft{1.7.10R2.8.1} [MFR Compat: Chococraft] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	UD	MineFactoryReloaded|CompatExtraBiomes{1.7.10R2.8.1} [MFR Compat: ExtraBiomes] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	UD	MineFactoryReloaded|CompatForestry{1.7.10R2.8.1} [MFR Compat: Forestry] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	UD	MineFactoryReloaded|CompatIC2{1.7.10R2.8.1} [MFR Compat: IC2] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	UD	MineFactoryReloaded|CompatProjRed{1.7.10R2.8.1} [MFR Compat ProjectRed] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	UD	MineFactoryReloaded|CompatRailcraft{1.7.10R2.8.1} [MFR Compat: Railcraft] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	UD	MineFactoryReloaded|CompatSufficientBiomes{1.7.10R2.8.1} [MFR Compat: Sufficient Biomes] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	UD	MineFactoryReloaded|CompatThaumcraft{1.7.10R2.8.1} [MFR Compat: Thaumcraft] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	UD	MineFactoryReloaded|CompatTwilightForest{1.7.10R2.8.1} [MFR Compat: TwilightForest] (MineFactoryReloaded-[1.7.10]2.8.1-174 (1).jar) 
	OpenModsLib class transformers: [stencil_patches:ENABLED],[movement_callback:ENABLED],[player_damage_hook:FINISHED],[map_gen_fix:FINISHED],[gl_capabilities_hook:ENABLED],[player_render_hook:ENABLED]
	Class transformer null safety: all safe
	AE2 Version: beta rv3-beta-6 for Forge 10.13.4.1448
	CoFHCore: -[1.7.10]3.1.4-329
	ThermalFoundation: -[1.7.10]1.2.6-118
	ThermalExpansion: -[1.7.10]4.1.5-248
	MineFactoryReloaded: -[1.7.10]2.8.1-174
	ForgeEssentials: Running ForgeEssentials 1.4.5.1341 (03a4404). Modules loaded: Afterlife, Backups, Chat, Commands, Economy, JScripting, Multiworld, Permissions, PlayerLogger, Protection, Remote, SignTools, Teleport, Tickets, WEIntegrationTools, WorldBorder, factions, mapper
	Mantle Environment: Environment healthy.
	TConstruct Environment: Environment healthy.
	NetherOres: -[1.7.10]2.3.1-22
	ThermalDynamics: -[1.7.10]1.2.1-172
	List of loaded APIs: 
		* ae2wct|API (1.7.10-rv3-1.8.6b) from WirelessCraftingTerminal-1.7.10-rv3-1.8.7.4b.jar
		* appliedenergistics2|API (rv3) from appliedenergistics2-rv3-beta-6.jar
		* Baubles|API (1.0.1.10) from ThermalFoundation-[1.7.10]1.2.6-118.jar
		* BetterQuesting|API (2.6) from BetterQuesting-2.0.233.jar
		* BloodMagicAPI (1.3.3-13) from BloodMagic-1.7.10-1.3.3-17.jar
		* BotaniaAPI (76) from Botania r1.8-249.jar
		* BuildCraftAPI|blocks (1.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|blueprints (1.5) from buildcraft-7.1.21.jar
		* BuildCraftAPI|boards (2.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|core (1.0) from extrautilities-1.2.12.jar
		* BuildCraftAPI|crops (1.1) from buildcraft-7.1.21.jar
		* BuildCraftAPI|events (2.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|facades (1.1) from buildcraft-7.1.21.jar
		* BuildCraftAPI|filler (4.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|fuels (2.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|gates (4.1) from buildcraft-7.1.21.jar
		* BuildCraftAPI|items (1.1) from buildcraft-7.1.21.jar
		* BuildCraftAPI|library (2.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|lists (1.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|power (1.3) from buildcraft-7.1.21.jar
		* BuildCraftAPI|recipes (3.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|robotics (3.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|statements (1.1) from buildcraft-7.1.21.jar
		* BuildCraftAPI|tablet (1.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|tiles (1.2) from buildcraft-7.1.21.jar
		* BuildCraftAPI|tools (1.0) from buildcraft-7.1.21.jar
		* BuildCraftAPI|transport (4.1) from buildcraft-7.1.21.jar
		* CarpentersBlocks|API (3.3.7) from Carpenters-Blocks-Mod-1.7.10.jar
		* CoFHAPI (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|block (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|core (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|energy (1.7.10R1.0.13B1) from extrautilities-1.2.12.jar
		* CoFHAPI|fluid (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|inventory (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|item (1.7.10R1.0.3) from EnderIO-1.7.10-2.2.8.381.jar
		* CoFHAPI|modhelpers (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|tileentity (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|transport (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHAPI|world (1.7.10R1.3.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|audio (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|gui (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|gui|container (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|gui|element (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|gui|element|listbox (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|gui|slot (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|inventory (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|render (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|render|particle (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|util (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|util|helpers (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|util|position (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|world (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* CoFHLib|world|feature (1.7.10R1.2.1) from CoFHCore-[1.7.10]3.1.4-329.jar
		* DraconicEvolution|API (1.2) from Draconic-Evolution-1.7.10-1.0.2h.jar
		* EnderIOAPI (0.0.2) from EnderIO-1.7.10-2.2.8.381.jar
		* EnderIOAPI|Redstone (0.0.2) from EnderIO-1.7.10-2.2.8.381.jar
		* EnderIOAPI|Tools (0.0.2) from EnderIO-1.7.10-2.2.8.381.jar
		* Forge-FEHooks (1448.2) from forgeessentials-1.7.10-1.4.6.1341-server.jar
		* ForgeEssentialsAPI (2.0) from forgeessentials-1.7.10-1.4.6.1341-server.jar
		* ForgeEssentialsAPI|Economy (2.0) from forgeessentials-1.7.10-1.4.6.1341-server.jar
		* ForgeEssentialsAPI|Perms (2.0) from forgeessentials-1.7.10-1.4.6.1341-server.jar
		* ForgeEssentialsAPI|Remote (2.0) from forgeessentials-1.7.10-1.4.6.1341-server.jar
		* Galacticraft API (1.1) from GalacticraftCore-1.7-3.0.12.501.jar
		* Guide-API|API (1.7.10-1.0.1-20) from Guide-API-1.7.10-1.0.1-20.jar
		* McJtyLib (1.8.1) from mcjtylib-1.8.1.jar
		* MekanismAPI|core (9.0.0) from Mekanism-1.7.10-9.1.0.281.jar
		* MekanismAPI|energy (9.0.0) from Mekanism-1.7.10-9.1.0.281.jar
		* MekanismAPI|gas (9.0.0) from Mekanism-1.7.10-9.1.0.281.jar
		* MekanismAPI|infuse (9.0.0) from Mekanism-1.7.10-9.1.0.281.jar
		* MekanismAPI|laser (9.0.0) from Mekanism-1.7.10-9.1.0.281.jar
		* MekanismAPI|reactor (9.0.0) from Mekanism-1.7.10-9.1.0.281.jar
		* MekanismAPI|recipe (9.0.0) from Mekanism-1.7.10-9.1.0.281.jar
		* MekanismAPI|transmitter (9.0.0) from Mekanism-1.7.10-9.1.0.281.jar
		* MekanismAPI|util (9.0.0) from Mekanism-1.7.10-9.1.0.281.jar
		* OpenBlocks|API (1.1) from OpenBlocks-1.7.10-1.6.jar
		* StorageDrawersAPI (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* StorageDrawersAPI|config (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* StorageDrawersAPI|event (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* StorageDrawersAPI|inventory (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* StorageDrawersAPI|pack (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* StorageDrawersAPI|registry (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* StorageDrawersAPI|render (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* StorageDrawersAPI|storage (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* StorageDrawersAPI|storage-attribute (1.7.10-1.2.0) from StorageDrawers-1.7.10-1.10.9.jar
		* WailaAPI (1.2) from Waila-1.5.10_1.7.10.jar
	AE2 Integration: IC2:OFF, RotaryCraft:OFF, RC:OFF, BuildCraftCore:ON, BuildCraftTransport:ON, BuildCraftBuilder:ON, RF:ON, RFItem:ON, MFR:ON, DSU:ON, FZ:OFF, FMP:ON, RB:OFF, CLApi:OFF, Waila:ON, Mekanism:ON, ImmibisMicroblocks:OFF, BetterStorage:OFF, OpenComputers:OFF, PneumaticCraft:OFF
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Player Count: 1 / 20; [EntityPlayerMP['kubajuk'/1593, l='world', x=-186,20, y=53,00, z=135,87]]
	Is Modded: Definitely; Server brand changed to 'fml,forge'
	Type: Dedicated Server (map_server.txt)
```

**Comment by brandon3055** [comment]
That crash is was not thrown by DE. It looks like its between iguana tweaks and thermal dynamics. Best bet to fix this is just update to 1.12 xD

**Comment by Tahakk** [comment]
I'll suggest you to try to go at [IguanaTweaks](https://github.com/SlimeKnights/IguanaTweaksTConstruct/issues) git since TD 1.7 is not developed anyway.
am I right? @kinglemming
